generics/constants/api-responses.js

```diff
@@ -306,4 +306,5 @@ module.exports = {
 	RELATED_ORGS_UPDATED_SUCCESSFULLY: 'Related orgs updated successfully',
 	ORG_EXTENSION_ALREADY_EXISTS: 'Organization extension already exists',
 	INVALID_MAPPING_DATA: 'Invalid mapping data',
+	LINK_IS_NOT_ACTIVE_YET: 'The link has not started yet. Please try again from: ',
```

Comment:
@borkarsaish65 Is this is the message they are expecting? do we have to confirm with products?